src/Files.App/App.xaml

```diff
@@ -37,7 +37,9 @@
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
 				<!--  Styles for the custom icons  -->
+				<ResourceDictionary Source="ms-appx:///UserControls/ThemedIcon/ThemedIcon.xaml" />
 				<ResourceDictionary Source="ms-appx:///Styles/PathIcons.xaml" />
+				<ResourceDictionary Source="ms-appx:///Styles/ThemedIconSet.xaml" />
```

Comment:
What does "Set" mean in this context?

Reply:
I think Styles/ThemedIcon should make more sense.
'Set' means 'a set of ThemedIcon styles' I assume.

Reply:
Are you okay with `ThemedIcons`?

Reply:
ThemedIcons works, as long as it doesn't cause any confusion with ThemedIcon.xaml which is the control's template

Reply:
We only use this in App.xaml.

Reply:
> We only use this in App.xaml. If we want consistency with other controls such as `SidebarStyles` and `GridSplitterStyles`, we might want to rename it to `ThemedIconStyles.xaml`.

Styles are technically what they are, but in terms of UI logic, they are different ThemedIcons

Reply:
file name isn't matter when it comes to developing once decided thus I'd like to keep consisntency here.

Reply:
File name doesn't matter when it comes to developing once decided thus I'd like to keep consisntency here.

Reply:
> File name doesn't matter when it comes to developing once decided thus I'd like to keep consisntency here.

Sure, but as long as people know where to go when adding icons in the future.

Reply:
I renamed to `ThemedIconStyles.xaml`.